extensions/ql-vscode/src/variant-analysis/variant-analysis-manager.ts

```diff
@@ -383,20 +394,22 @@ export class VariantAnalysisManager
       );
     }
 
-    const queryText = await readFile(queryFile, "utf8");
+    const queryText = await readFile(firstQueryFile, "utf8");
 
-    // TODO: Once we have basic support multiple queries, and qlPackDetails has
-    // more than 1 queryFile, we should set this to have a proper value
-    // (e.g. { language: variantAnalysisLanguage })
-    const queries: VariantAnalysisQueries | undefined = undefined;
+    const queries: VariantAnalysisQueries | undefined =
+      qlPackDetails.queryFiles.length === 1
+        ? undefined
+        : {
```

Comment:
I'm not suggesting we do anything more in this PR, but I think now would be a good time to think more about this `queryFiles` field. I'll write up some of my thoughts in an issue.